src/Pages/addPhoto/AddFramePhoto.js

```diff
@@ -87,50 +88,50 @@ class AddPhoto extends React.Component {
     render(){
         const {isUpload , isDelete , frameUrl } = this.state;
         return(
-            <div className='addFramePhoto-body'>
-                <div className='addFramePhoto-header'>
-                    <label className='addFramePhoto-header-text'><strong>Add Frame Photo</strong></label>
-                </div>
-                <div className='addFramePhoto-outter-tag'>              
-                    <div className='frame-img-tag'>
-                        <input 
-                            type="file" 
-                            name="uploadfile" 
-                            onChange={this.uploadFrameImg} 
-                            id="frameimg" 
-                            style={{display:'none'}}
-                        />
-                        <input 
-                            type="button" 
-                            onClick={this.deleteImg} 
-                            id="framedel" 
-                            style={{display:'none'}}
-                        />  
-                        <div className='showFrame-img-tag'>
-                            <img src={frameUrl || 'https://via.placeholder.com/400x300'} alt="Upload logo" className={frameUrl ? 'frameImg-tag' : 'defaultFrame-img-tag'} />
-                        </div>                      
-                        <div className='btn-tag'>
-                                 <Button 
-                                    className='uploadFrame-btn' 
-                                    variant="contained" 
-                                    color="primary"
-                                    startIcon={<AddIcon />}
-                                >
-                                    <label className='btn-text-css' htmlFor="frameimg">Add</label>
-                                </Button>
-                               <Button 
-                                    className='uploadFrame-btn' 
-                                    variant="contained" 
-                                    color="secondary"
-                                    startIcon={<DeleteIcon />}
-                                >
-                                    <label className='btn-text-css' htmlFor='framedel' >Delete</label>
-                                </Button>
+            <div>
```

Comment:
You can use Fragment to wrap multiple components if you are not assigning any className to it.